controllers/movies_controller.js

```diff
@@ -0,0 +1,55 @@
+var Movie = require("../models/movie")
+var Rental = require("../models/rental")
+
+var MoviesController = {
+
+  index: function(req, res, next) {
+    Movie.all(function(error, movies) {
+      if(error) {
+        var err = new Error("Error retrieving movies:\n" + error.message)
+        err.status = 500
+        next(err)
+      } else {
+        res.json(movies)
+      }
+    })
+  },
+
```

Comment:
I am a little fuzzy on my javascript but I think you want to make sure to have semicolons at the end of each line. Looks like this is something to lookout for across your app.